includes/suppress-ads.php

```diff
@@ -5,6 +5,8 @@
  * @package Newspack
```

Comment:
There's an instance of the legacy `Newspack_Ads` class name instead of `Core` on line 33 here. This causes a fatal error to be thrown on the block editor:

```
PHP Fatal error:  Uncaught Error: Cannot use object of type WP_Error as array in /Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/includes/wizards/class-advertising-wizard.php:563
```

Reply:
Fixed by dbe1710a54dfdf0f10af2abcf1cc90461c937509

Reply:
Confirmed, thanks!